GitUI/UserControls/RevisionGrid/RevisionDataGridView.cs

```diff
@@ -330,8 +330,18 @@ public void Clear()
         {
             _backgroundScrollTo = 0;
 
-            // Force the background thread to be killed, we need to be sure no background processes are running. Not the best practice, but safe.
-            _backgroundThread.Abort();
```

Comment:
This will help with .NET Core migration.
https://github.com/gitextensions/gitextensions/pull/7162#discussion_r329353754